Protocol.md

```diff
@@ -154,7 +154,7 @@ ReactiveSocket frames begin with a ReactiveSocket Frame Header. The general layo
 ```
 
 * __Stream ID__: (31 bits = max value 2^31-1 = 2,147,483,647) Unsigned 31-bit integer representing the stream Identifier for this frame or 0 to indicate the entire connection.
-  * Transport protocols that include demultiplexing, such as HTTP/2, MAY omit the Stream ID field if all parties agree. The means of negotiation and agreement is left to the transport protocol. 
+  * Transport protocols that include demultiplexing, such as HTTP/2, MAY omit the Stream ID field if all parties agree. The means of negotiation and agreement is left to the transport protocol.
```

Comment:
Is there a completely unambiguous and implementable public definition of what we mean by HTTP/2 support?

a) quasi-TCP over a single HTTP/2 stream
b) HTTP/2 extension (https://github.com/ReactiveSocket/reactivesocket/issues/84)
https://github.com/ReactiveSocket/reactivesocket/issues/84

This sentence seems to imply the latter implementation.